### PR TITLE
Remove material-ui dependency

### PR DESCRIPTION
**Pull request by sidharthmehra**
- Removed Material-ui dependency from Row Select Table.
- Added React-Simple-Dropdown and styled as per styleguide.
- Added Basic input and button fields.
- Added Custom Loader Component to replace material-ui circular progress.

Working example in code sandbox:
https://codesandbox.io/s/w2z57m9045

Pending stuff:
- Need to anchor the dropdown menu above the button, this is needed as the table would end up going to the bottom of the screen and users would be unable to select their options.

### CODE REVIEW

**Comment by sidharthmehra** [comment]
@rahulrcopart @colshacol @larrylicopart please review.

File: package.json [hunk 1]
```diff
@@ -11,8 +11,10 @@
     "prepublish": "yarn build"
   },
   "dependencies": {
-    "classnames": "^2.2.5",
-    "prop-types": "^15.5.10"
+    "classnames": "2.2.6",
+    "prop-types": "15.6.2",
+    "react-md-spinner": "0.3.0",
```
Comment (rahulrcopart) [inline package.json]: I checked this package , it has material-ui as dependancy

Comment (rahulrcopart) [reply package.json]: So material-ui became a transitive dependancy instead of a direct one

Comment (sidharthmehra) [reply package.json]: Good catch, just pushed a custom Loader component that does not use Fabric/Material.
Thanks @colshacol  for working on this!

**Comment by sidharthmehra** [comment]
New updated sandbox:
https://codesandbox.io/s/l951761vrl

**Pull request merged**